cuegui/cuegui/FrameMonitor.py

```diff
@@ -428,14 +428,14 @@ def _filterStatusSetup(self, layout):
         btn.setMenu(menu)
         menu.triggered.connect(self._filterStatusHandle)  # pylint: disable=no-member
 
-        for item in [("Clear", QtCore.Qt.ALT + QtCore.Qt.Key_QuoteLeft),
+        for item in [("Clear", QtCore.Qt.ALT | QtCore.Qt.Key_QuoteLeft),
```

Comment:
Just checking are we sure about the syntax here? Not an expert but this looks like a switch from "and" to "or".

Reply:
When using PySide6 I get these warnings : 
```
OpenCue/cuegui/cuegui/FrameMonitor.py:438: UserWarning: 
The "+" operator is deprecated in Qt For Python 6.0 .
Please use "|" instead.
  ("Eaten", QtCore.Qt.ALT + QtCore.Qt.Key_6)]:
```

Reply:
Ok cool, yeah looks like it's the right choice. Just looked off at first glance :)

Reply:
The change from `+` to `|` is indeed the correct approach for combining Qt key modifiers and key codes. This ensures that the combined value correctly represents both the modifier and the key, allowing Qt to interpret the input as intended.